cl/people_db/models.py

```diff
@@ -626,22 +626,28 @@ class Position(AbstractDateTimeModel):
         ("rec_postpone", "Recommendation Postponed"),
         ("rec_bad", "Recommended Unfavorably"),
     )
+    ELECTION_PARTISAN = "e_part"
+    ELECTION_NON_PARTISAN = "e_non_part"
+    APPOINTMENT_PRESEDENT = "a_pres"
```

Comment:
Sorry, I know this is my mistake, but can you fix this typo before it becomes part of the legacy we leave our children?